### ISSUE TITLE
xvfb tests fail on Mac OS

### ISSUE THREAD

**Issue by satra** [nipy/nipype#2263]
### Summary
nipype tests should skip Xvfb related tests on Mac OS if not available

### Actual behavior
```
                raise RuntimeError(
>                   'Xvfb requires root permissions to run in OSX. Please '
                    'make sure that an X server is listening and set the '
                    'appropriate config on either $DISPLAY or nipype\'s '
                    '"display_variable" config. Valid X servers include '
                    'VNC, XQuartz, or manually started Xvfb.')
E               RuntimeError: Xvfb requires root permissions to run in OSX. Please make sure that an X server is listening and set the appropriate config on either $DISPLAY or nipype's "display_variable" config. Valid X servers include VNC, XQuartz, or manually started Xvfb.
```

i've tried the following configurations:

1. XQuartz is running.
2. `DISPLAY` variable is unset

### Expected behavior

on platforms where we expect issues, we should ensure that tests are either skipped or users are informed as to what to do.

### How to replicate the behavior

Run test on Mac OS

### Script/Workflow details

pytest nipype/utils

### Platform details:

{'pkg_path': '/software/nipy-repo/nipype/nipype', 'commit_source': 'repository', 'commit_hash': 'fc2c5b4', 'nipype_version': '1.0.0-dev+gfc2c5b4', 'sys_version': '3.6.0 |Continuum Analytics, Inc.| (default, Dec 23 2016, 13:19:00) \n[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)]', 'sys_executable': '/software/miniconda3/envs/dev36pype/bin/python', 'sys_platform': 'darwin', 'numpy_version': '1.13.1', 'scipy_version': '0.19.0', 'networkx_version': '1.11', 'nibabel_version': '2.3.0dev', 'traits_version': '4.6.0'}
1.0.0-dev+gfc2c5b4

### Execution environment

- My python environment outside container

**Comment by oesteban** [comment]
Could you confirm that at least this one is passing?:

https://github.com/nipy/nipype/blob/master/nipype/utils/tests/test_config.py#L182-L195

**Comment by satra** [comment]
yes, that one passes.

**Comment by oesteban** [comment]
Fixed in #2363.
